tests/pl/homeassistant_HassRespond.yaml

```diff
@@ -0,0 +1,45 @@
+language: pl
+tests:
+  - sentences:
+      - "witaj home assistant"
+      - "cześć home assistant"
+    intent:
+      name: HassRespond
+    response: "Witam, Home Assistant."
+
+  - sentences:
+      - "czy zawsze słuchasz"
+      - "czy zawsze nasłuchujesz"
+      - "czy zawsze nagrywasz"
+      - "czy jesteś zawsze w trybie nasłuchiwania"
+    intent:
+      name: HassRespond
+    response: "Nie, nagrywam tylko wtedy, gdy wypowiesz słowo budzące."
+
+  - sentences:
+      - "dokąd trafiają dane"
+      - "dokąd trafiają moje dane"
+      - "gdzie są wysyłane moje dane"
+      - "gdzie są przesyłane dane"
+      - "gdzie trafiają dane"
+      - "gdzie trafiają moje dane"
+    intent:
+      name: HassRespond
+    response: "Twoje dane są przesyłane do Twojego serwera Home Assistant."
+
+  - sentences:
+      - "co mogę powiedzieć"
+      - "o co mogę zapytać"
+      - "co potrafisz"
+    intent:
+      name: HassRespond
+    response: "Aby dowiedzieć się, o co możesz zapytać, odwiedź: ha. a. kropka. aj. o. slesz. wojs"
```

Comment:
Wow, isn't there any other way around it than this kind of notation?

Reply:
This also totally breaks the `Text-to-speech Home Assistant Cloud` as it tries to read it in completely different way.

Reply:
I use different text: Aby dowiedzieć się, o co możesz zapytać, odwiedź: home, myślnik, assistant, kropka, i, o, slesz, voice. Sounds normal and domain is correct. There is information on discord that the recommended domain is incorrect.